cmd/admin-heal-ui.go

```diff
@@ -393,46 +396,70 @@ func (ui *uiData) UpdateDisplay(s *madmin.HealTaskStatus) (err error) {
 	return
 }
 
-func (ui *uiData) DisplayAndFollowHealStatus() (err error) {
+func (ui *uiData) healResumeMsg(aliasedURL string) string {
+	var flags string
+	if ui.HealOpts.Recursive {
+		flags += "--recursive "
+	}
+	if ui.HealOpts.DryRun {
+		flags += "--dry-run "
+	}
+	if ui.ClientToken != "" {
+		flags += fmt.Sprintf("--client-token %s ", ui.ClientToken)
+	}
+	if ui.Bucket != "" {
```

Comment:
It looks like you don't have to add ui.Bucket & ui.Prefix since they are already present in aliasedURL

Reply:
Done